src/components/PeoplePicker/PeoplePicker.md

```diff
@@ -0,0 +1,32 @@
+### Notes for use
+
+PeoplePicker is simply NormalPeoplePicker [exported from Fabric](https://developer.microsoft.com/en-us/fabric#/components/peoplepicker).
```

Comment:
Why mention the underlying implementation? Maybe describe when it should be used instead.

Reply:
The primary reason I didn't add a lengthy description is the link to the Fabric docs contains that information and specifics may become stale here. In the Callout pass-through component, we provide similar [limited description](https://github.com/Microsoft/YamUI/blob/master/src/components/Callout/Callout.md#notes-for-use). But I'll add additional general information about what the component should be used.

Reply:
since we're implementing this component backward (Engineering-focused instead of Design/UX-focused) think it makes sense to link out to Fabric's docs. but we should remove this when we dial it in and limit its props